lessons/codap-exploring/langs/en-us/index.adoc

```diff
@@ -122,7 +116,8 @@ Brainstorm other scenarios where Booleans are useful in and out of the programmi
 === Investigate
 [.lesson-instruction]
 --
-In pairs, complete @printable-exercise{pages/booleans.adoc}, making predictions about what a variety of Boolean expressions will return and testing them in the editor.
+- Turn to @printable-exercise{pages/booleans.adoc}.
+- Write down your predictions before testing them in CODAP. When you're ready, test the Boolean expression in @starter-file{boolean}. Again, you will work from left to right, clicking on the column name, and then choosing "Edit Formula" from the drop-down menu that appears. You will type each Boolean into the "Edit Formula" box.
```

Comment:
Please update the starter file so that it does not have a number 18. Also, do you mean to have the first column done already. If so, please update the directions on the workbook page to say that the first one has been done for you.